src/frontend/query_commands.ml

```diff
@@ -243,6 +243,13 @@ let reconstruct_identifier pipeline pos = function
         aux acc (succ i) in
     aux [] offset
 
+let keywords pipeline kinds =
+  match kinds with
+  | []
+  | _ :: _ when List.mem `Keywords ~set:kinds ->
```

Comment:
That's a bit weird. Why not make it an `if`?

Reply:
No longer present in the up to date patch